integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/TextElideTest.kt

```diff
@@ -25,9 +41,71 @@ import com.android.designcompose.annotation.DesignDoc
 @DesignDoc(id = "oQ7nK49Ya5PJ3GpjI5iy8d")
 interface TextElideTest {
     @DesignComponent(node = "#stage") fun MainFrame()
+
+    @DesignComponent(node = "#stage2") fun stage2()
+
+    @DesignComponent(node = "#stage3") fun stage3()
+
+    @DesignComponent(node = "#stage4") fun stage4()
+
+    @DesignComponent(node = "#stage5") fun stage5()
 }
 
 @Composable
 fun TextElideTest() {
-    TextElideTestDoc.MainFrame()
+    val list = mutableListOf("#stage", "#stage2", "#stage3", "#stage4", "#stage5")
+    val stageState = remember { mutableStateOf("#stage") }
+
+    val containerHeight = remember { mutableFloatStateOf(61f) }
+
+    when (stageState.value) {
+        "#stage" -> TextElideTestDoc.MainFrame()
+        "#stage2" -> TextElideTestDoc.stage2()
+        "#stage3" ->
+            TextElideTestDoc.stage3(modifier = Modifier.height(containerHeight.floatValue.dp))
+        "#stage4" ->
+            TextElideTestDoc.stage4(modifier = Modifier.height(containerHeight.floatValue.dp))
+        "#stage5" ->
+            TextElideTestDoc.stage5(modifier = Modifier.height(containerHeight.floatValue.dp))
+    }
+
+    Column(modifier = Modifier.offset(y = 360.dp)) {
+        list.forEach {
+            Row(modifier = Modifier.padding(4.dp), verticalAlignment = Alignment.CenterVertically) {
+                RadioButton(
+                    selected = (it == stageState.value),
+                    onClick = { stageState.value = it }
+                )
+                Text(text = it, modifier = Modifier.padding(start = 4.dp))
+            }
+        }
+
+        HorizontalDivider(modifier = Modifier.fillMaxWidth().padding(bottom = 4.dp))
```

Comment:
This `fillMaxWidth()` might be why the screenshot stretched to be so wide. I'm not sure what other consequences there are to @timothyfroehlich's screenshot changes -- if we don't want to use that yet you could try removing this.

Reply:
I like Tim's change to just capture the design doc. But right now there is one screenshot mismatch I haven't figured out what is happening.
I have run the `./gradlew recordRoborazziDebug` and uploaded new screenshots but the checks failed on the mask test.